### PR TITLE
Created pull request template

### PR DESCRIPTION
**Pull request by aditmehta9**
### Description
Created a pull request template in .github folder.

Fixes [#7](https://github.com/anitab-org/anitab-org.github.io/issues/7)

### Type of Change:
- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

### CODE REVIEW

**Comment by aditmehta9** [comment]
@mayburgos I have created a pull request, please review.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### Checklist:
+**Delete irrelevant options.**
+
+- [ ] My PR follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code or materials
+- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] Any dependent changes have been merged
+- [ ] I have written Kotlin Docs whenever is applicable
```
Comment (annabauza) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:35]: we don't use kotlin here

Comment (aditmehta9) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Sorry, I didn't notice. I didn't read whole template. This kind of mistake won't happen again.

**Comment by aditmehta9** [comment]
@annabauza I have updated the file.

**Comment by aditmehta9** [comment]
> 
> 
> small copy/paste mistake

Sorry, I hadn't noticed.

**Pull request merged**